roles/pipeline/pipeline_example/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Can you remove the empty line at beginning?

Reply:
done